### ISSUE TITLE
usr_uptime.lua - shows the wrong users uptime

### ISSUE THREAD

**Issue by Sopor** [luadch/luadch#39]
When trying to get a users uptime by right clicking the username in the user list it will only show my own uptime instead of the users. This is the command: `+useruptime CT2 <username>`.

**Comment by HypoM** [comment]
I've been poking on this a bit - and it seems that the regexp on line 193 is the culprit.  
`param1` and `param2` always ends up being empty, even if you use the commands `+useruptime CT2 <username>` and `+useruptime CT1 <username>`.  
  
I think it might be a better solution to change line 193 so it splits instead, and on line 194 (the if-statement) we check how many parts we get (if 0, run that, if more show usage).  
That way we are sure later in the script that both `param1` and `param2` will contain something.

**Comment by execut3r** [comment]
I've changed the line 193 on our hub from
` local param1, param2 = utf_match( parameters, "^?(%S+)** (%S+)$" )`

to:
`local param1, param2 = utf_match( parameters, "^(%S+) (%S+)$" )`

This seems to work, but i have no clue if that change is correct. At least i can finally check for users loging in and out without getting any time in the hub.

**Comment by 4C3T** [comment]
Seem like that change was done to the compilation I have.
In my case it's not working at all. Running Linux Mint

I'm only getting this:
`=== USER UPTIME ========================================================================

        Uptime stats about:  xyzUser

	YEAR		MONTH		UPTIME
	--------------------------------------------------------------------------------------------------------------------------------------------

	2020	December	0 days, 0 hours, 0 minutes, 0 seconds

======================================================================== USER UPTIME ===`

It's still showing the wrong username... That was my account it tried to fetch info about.

I kind of like this hub, but it's somewhat complex to be honest.
It's at the same time very nice that it doesn't have compiled crap which can't be altered.

**Comment by 4C3T** [comment]
Just randomly tested !uptime, I know, unrelated, maybe, maybe not...
But, I got some very weird numbers...

"Hub Uptime (complete): 0 years, 227 days, 3 hours, 38 min, 56 sec
Hub uptime (session) 0 years,  4 hours, 36 min, 17 sec."

That last entry was pretty accurate, but "227 days"??? Where the crap-biscuit did it get that from???
It' been running for like 2 days now, tops.
Is it perhaps something stored in a file somewhere or what?
I did download an already compiled version of this hub...

Edit: Just figured out that this release (from here) probably was tested, or had a short run prior, some 227 days ago.
Just entered !runtime reset so it's zeroed now
